mkdocs_src/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Remove file.

Reply:
@alitariq4589 You didn't remove this

Reply:
I removed `mkdocs_src/.vscode` file now. Actually it was hidden so I could not see it last time.